hadoop-ozone/dist/src/main/compose/test-all.sh

```diff
@@ -19,17 +19,22 @@
 #
 # Test executor to test all the compose/*/test.sh test scripts.
 #
-
 SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )
 ALL_RESULT_DIR="$SCRIPT_DIR/result"
-
+PROJECT_DIR="$SCRIPT_DIR/.."
 mkdir -p "$ALL_RESULT_DIR"
-rm "$ALL_RESULT_DIR/*"
+rm "$ALL_RESULT_DIR/*" || true
+
+if [ "$OZONE_WITH_COVERAGE" ]; then
+   java -cp "$PROJECT_DIR"/share/coverage/$(ls "$PROJECT_DIR"/share/coverage | grep test-util):"$PROJECT_DIR"/share/coverage/jacoco-core.jar org.apache.hadoop.test.JacocoServer &
+   DOCKER_BRIDGE_IP=$(docker network inspect bridge --format='{{(index .IPAM.Config 0).Gateway}}')
```

Comment:
Some of the compose environments have a dedicated network.  I think those would have problems starting SCM:

```
hadoop-ozone/dist/src/main/compose/ozone-topology/docker-compose.yaml
hadoop-ozone/dist/src/main/compose/ozonesecure-om-ha/docker-compose.yaml
hadoop-ozone/dist/src/main/compose/ozonesecure-mr/docker-compose.yaml
```

(I'm not sure because I have trouble starting any of the environments with coverage enabled.)

Reply:
> (I'm not sure because I have trouble starting any of the environments with coverage enabled.)

How did you try? Docker bridge ip supposed to be available on all the containers. But can be wrong on OSX. But worked well on github environments.

Reply:
IP is available, can be pinged, but connection to port is refused.  Confirmed that Jacoco server is running on localhost:6300.

I'm [experimenting](https://github.com/adoroszlai/hadoop-ozone/commits/HDDS-3757) with running Jacoco in its own container.